opentelemetry/src/main/scala/zio/telemetry/opentelemetry/Tracing.scala

```diff
@@ -17,8 +18,9 @@ object Tracing {
   trait Service {
     private[opentelemetry] def currentNanos: UIO[Long]
     private[opentelemetry] val currentSpan: FiberRef[Span]
-    private[opentelemetry] def createRoot(spanName: String, spanKind: Span.Kind): UIO[Span]
-    private[opentelemetry] def createChildOf(parent: Span, spanName: String, spanKind: Span.Kind): UIO[Span]
+    private[opentelemetry] def createRoot(spanName: String, spanKind: Span.Kind): UManaged[Span]
+    private[opentelemetry] def createChildOf(parent: Span, spanName: String, spanKind: Span.Kind): UManaged[Span]
+    private[opentelemetry] def end(span: Span): UIO[Unit]
```

Comment:
If Spans are managed, why do we need to expose `end`?

Reply:
Good point! We don't need to indeed

Fixed with https://github.com/zio/zio-telemetry/pull/136/commits/5189adc8ba1a577ab76892b81bfa834ce9c70551